projects/demo/src/modules/app/version-manager/version-manager.component.ts

```diff
@@ -15,15 +15,7 @@ import {TUI_VERSIONS_META_OPTIONS} from './versions.constants';
 @Component({
     standalone: true,
     selector: 'version-manager',
-    imports: [
-        FormsModule,
-        NgForOf,
-        NgIf,
-        TuiDataList,
-        TuiSelectModule,
-        TuiStringifyContentPipe,
-        TuiStringifyPipe,
-    ],
+    imports: [FormsModule, NgForOf, TuiButton, TuiChevron, TuiDataList, TuiSelectModule],
```

Comment:
Looks like we do not need forms and select here

Reply:
fixed